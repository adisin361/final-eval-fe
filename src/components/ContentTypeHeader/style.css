@import url('https://fonts.googleapis.com/css2?family=Montserrat:wght@300;500;700&display=swap');

.content-type-header {
  color: rgba(0, 0, 0, 0.748);
  display: flex;
  align-items: center;
  height: 6vh;
  background-color: white;
  width: 85vw;
  padding-left: 30px;
  font-family: Montserrat;
  font-weight: bolder;
  font-size: 1.3rem
}