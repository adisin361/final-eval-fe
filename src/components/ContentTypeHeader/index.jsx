import React from 'react';
import './style.css';

const ContentTypeHeader = () => {
  return (
    <div className="content-type-header">
      <p>Content Types</p>
    </div>
  );
};

export default ContentTypeHeader;