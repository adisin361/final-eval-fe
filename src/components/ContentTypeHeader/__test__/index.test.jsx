import React from 'react';
import ContentTypeHeader from '..';
import { render } from '@testing-library/react';

describe('Content Type Header', () => {
  it('should render correctly', () => {
    const { asFragment } = render(<ContentTypeHeader />);
    expect(asFragment()).toMatchSnapshot();
  });
});