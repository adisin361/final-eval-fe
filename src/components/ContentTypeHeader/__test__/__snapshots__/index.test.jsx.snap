// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Content Type Header should render correctly 1`] = `
<DocumentFragment>
  <div
    class="content-type-header"
  >
    <p>
      Content Types
    </p>
  </div>
</DocumentFragment>
`;
